script.js

```diff
@@ -0,0 +1,298 @@
+body.prepend(div);
+const bodyDiv = document.querySelector('div');
+bodyDiv.insertAdjacentHTML('afterbegin', textArea);
+bodyDiv.insertAdjacentHTML('beforeend', '<div class = "keyboard" ></div>')
+
+function isLang() {
+    return (lang === 'Eng') ? keyboardEng : keyboardRus;
+}
+
+isLang().reverse().map(el => {
+    bodyDiv.children[1].insertAdjacentHTML('afterbegin', `<div class = "keyboard${lang} ${el[0]} ${el[3]} "> <p class = "little_letter">${el[1]}</p> <p class = "big_letter letter_none">${el[2]}</p></div>`);
+});
+
+const keyboard = document.querySelector('.keyboard');
+const keyboardDOM = Array.from(document.querySelectorAll(`.keyboard${lang}`));
+let textAreaIn = document.querySelector('.textArea');
+const little_letter = Array.from(document.querySelectorAll('.little_letter'));
+const big_letter = Array.from(document.querySelectorAll('.big_letter'));
+const capslock = document.querySelector('.capslock');
+
+let key;
+let keyCode;
+let keyLower;
+let left = 'left';
+let right = 'right';
+let state;
+let iter = true;
+
+//------------------------------- keydown ----------------------------------
+window.addEventListener('keydown', (event) => {
+    state = 'add';
+    key = event.key;
+    keyCode = event.code;
+    keyLower = key.toLowerCase();
+
+    event.preventDefault();
+
+    simpleAndSpecial(state);
+    switchKeyCode(state);
+    (sessionStorage.hasOwnProperty('ctrl') && sessionStorage.hasOwnProperty('alt')) ? replaceLanguage() : undefined;
+})
+
+//------------------------------- keyup ----------------------------------
+window.addEventListener('keyup', (event) => {
+    state = 'remove';
+    key = event.key;
+    keyCode = event.code;
+    keyLower = key.toLowerCase();
+
+    simpleAndSpecial(state);
+    switchKeyCode(state);
+});
+
+//------------------------------- end listener ----------------------------------
+
+function replaceLanguage() {
+    if (sessionStorage.hasOwnProperty('lang')) {
+        (sessionStorage.hasOwnProperty('ctrl') && sessionStorage.hasOwnProperty('alt')) ? ((sessionStorage.getItem('lang') === 'Eng') ? sessionStorage.setItem('lang', 'Rus') : sessionStorage.setItem('lang', 'Eng')) :
+            undefined;
+        window.location.reload();
+
+    } else {
+        (sessionStorage.hasOwnProperty('ctrl') && sessionStorage.hasOwnProperty('alt')) ? ((localStorage.getItem('lang') === 'Eng') ? sessionStorage.setItem('lang', 'Rus') : sessionStorage.setItem('lang', 'Eng')) :
+            undefined;
+        window.location.reload();
+    }
+}
+
+function switchKeyCode(state) {
+
+    switch (keyCode) {
+
+        case 'ShiftLeft':
+            keySide(keyLower, left, state);
+            break;
+        case 'ShiftRight':
+            keySide(keyLower, right, state);
+            break;
+        case 'ControlLeft':
+            keySide(keyLower, left, state);
+            break;
+        case 'ControlRight':
+            keySide(keyLower, right, state);
+            break;
+        case 'AltLeft':
```

Comment:
If you have the same callback for several cases you can use
``` javascript
case 'ShiftLeft':
case 'ControlLeft':
case 'AltLeft':
    keySide(keyLower, left, state);
```
Use also `case dafault` if you can.